kmir/src/kmir/kmir.py

```diff
@@ -64,6 +64,7 @@ def kcfg_explore(self, label: str | None = None) -> Iterator[KCFGExplore]:
             llvm_definition_dir=self.llvm_library_dir,
             bug_report=self.bug_report,
             id=label if self.bug_report is not None else None,  # NB bug report arg.s must be coherent
+            interim_simplification=200,  # working around memory problems in LLVM backend calls
```

Comment:
This `interim-simplification` option was added to the server to work around a problem with `symbolic-args-fail.main`. Without adding that, the server would grow an enormous memory footprint during the proof. I experimented with the request in question but could not find out where exactly the memory is allocated. It is _not_ Haskell heap data, so probably the LLVM backend allocates it somehow.